lib/git_reflow.rb

```diff
@@ -27,12 +27,11 @@ def status(destination_branch)
     pull_request = git_server.find_open_pull_request( :from => current_branch, :to => destination_branch )
 
     if pull_request.nil?
-      puts "\n[notice] No pull request exists for #{current_branch} -> #{destination_branch}"
-      puts "[notice] Run 'git reflow review #{destination_branch}' to start the review process"
+      say "\nNo pull request exists for #{current_branch} -> #{destination_branch}", :notice
+      say "Run 'git reflow review #{destination_branch}' to start the review process", :notice
     else
-      puts "Here's the status of your review:"
+      say "Here's the status of your review:"
```

Comment:
Migration from `puts` => `say` is great! But will we still be migrating from `say` to using a `logger` like: https://github.com/reenhanced/gitreflow/issues/156 ?

Seems like changing something that will be changed anyways, maybe its a good time to include those changes here!